lib/language_server/protocol/constant.rb

```diff
@@ -36,42 +36,6 @@ module Constant
       autoload :TokenFormat, "language_server/protocol/constant/token_format"
       autoload :UniquenessLevel, "language_server/protocol/constant/uniqueness_level"
       autoload :WatchKind, "language_server/protocol/constant/watch_kind"
-
-      require "language_server/protocol/constant/code_action_kind"
```

Comment:
If I can remember correctly there should be a reason behind this `require` 🤔

Reply:
I remembered. I wanted to `require` (eager load) everything with just requiring single file but `autoload` was also needed  since some files may have dependencies. So actually both require and autoload are needed.

I'm going to close this PR though, thank you for your interest.